assets/js/modules/search-console/dashboard/dashboard-widget-sitestats.js

```diff
@@ -103,7 +103,6 @@ class SearchConsoleDashboardWidgetSiteStats extends Component {
 			options.vAxis.textPosition = 'none';
 			options.vAxis.gridlines.color = '#fff';
 			options.vAxis.minorGridlines.color = '#fff';
-			options.vAxes = {};
```

Comment:
Note: removing these doesn't clutter up the UI, because Google Charts is smart enough to hide the vAxis labels if there's more than two.